mineral-ui/mineral-ui
# [Grid] New components

### Description

- Add Grid & GridItem components
- Add demos & examples
- Add specs and snapshots
- Refactor Flex to share some things with Grid & GridItem
- Update Flex's gutter example to match Grid's and update Box's intro to include Grid & GridItem
- Fix display issue with Layout examples and update snapshots

### Motivation and context

Closes #148

### Screenshots, videos, or demo, if appropriate


https://148-grid--mineral-ui.netlify.com/components/grid

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to [Grid](https://148-grid--mineral-ui.netlify.com/components/grid) & [GridItem](https://148-grid--mineral-ui.netlify.com/components/grid-item)
3. Confirm all looks well

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist




### How does this PR make you feel?

![Tron grid and lightcycles](https://user-images.githubusercontent.com/486540/38888073-d740b9d4-4237-11e8-8d96-3de044568eca.gif)
